internal/config/config.go

```diff
@@ -19,6 +19,7 @@ import (
 	"github.com/influxdata/telegraf/plugins/serializers"
 
 	"github.com/influxdata/config"
+	"github.com/naoina/toml"
```

Comment:
Can you remove the naoina/toml dependency here? I believe there is a function in influxdata/config that you can use to load and parse the toml file